server/genserver.go

```diff
@@ -102,25 +103,25 @@ func (s *Server) Serve() error {
 	ingestURL := os.Getenv("TRACING_INGEST_URL")
 	isLocal := os.Getenv("_IS_LOCAL") == "true"
 	if (tracingToken != "" && ingestURL != "") || isLocal {
-		// Add rate limited sampling. We will only sample [Param] requests per second
-		// and [MaxOperations] different endpoints. Any endpoint above the [MaxOperations]
-		// limit will be probabilistically sampled.
-		cfgSampler := &jaegercfg.SamplerConfig{
-			Type:          jaeger.SamplerTypeRateLimiting,
-			Param:         5,
-			MaxOperations: 100,
-		}
-		cfgTags := []opentracing.Tag{
-			opentracing.Tag{Key: "app_name", Value: os.Getenv("_APP_NAME")},
-			opentracing.Tag{Key: "build_id", Value: os.Getenv("_BUILD_ID")},
-			opentracing.Tag{Key: "deploy_env", Value: os.Getenv("_DEPLOY_ENV")},
-			opentracing.Tag{Key: "team_owner", Value: os.Getenv("_TEAM_OWNER")},
+		samplingRate := .05 // 5%% of requests
```

Comment:
Is there anyway we could add an upper bound here? Since we're in the early stages I'm concerned our SignalFX gateway won't be able to handle this increased throughput.

Reply:
Pending feedback from @mohit I'm just going to revert this commit since I have no context on the state of the gateway and the docs around this sampler do not make me confident that I have configured it correctly 🤷‍♂

Reply:
moved to https://github.com/Clever/wag/pull/249